.github/CODEOWNERS

```diff
@@ -104,3 +104,4 @@
 /configs/default/WIZZ-* @bnn1044 @betaflight/unified-target-administrators
 /configs/default/YUPF-* @Faduf @betaflight/unified-target-administrators
 /configs/default/ZEEZ-* @Guidus93 @betaflight/unified-target-administrators
+/configs/default/SPBE-* @SEA729 @betaflight/unified-target-administrators
```

Comment:
Remove as `CODEOWNERS` file is maintained by Betaflight.

Reply:
thanks for your review